spec/cypress/support/step_definitions/common-navigation.js

```diff
@@ -41,6 +42,12 @@ Given("I am on {string} page", (page) => {
   cy.visit(path);
 });
 
+Given("I am on {string} page with token {string}", (page, token) => {
+  const path = pagePaths[page];
+  const pathWithToken = `${path}?token=${token}`;
+  cy.visit(pathWithToken);
+});
```

Comment:
What does this achieve vs just adding the token into the URL in the path definition above?

Reply:
Done